spring-cloud-azure-context/pom.xml

```diff
@@ -28,5 +28,12 @@
             <groupId>com.microsoft.azure</groupId>
             <artifactId>azure</artifactId> <!-- We may not need include all azure management sdks in a batch -->
         </dependency>
+
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.6</version>
+            <scope>compile</scope>
```

Comment:
Can we move this dependency to the bom `spring-cloud-azure-dependencies` and remove the version from here?

Reply:
good point, will refine it.